src/Navigation/Navigation.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
 import { NavigationContainer } from '@react-navigation/native';
-import { createStackNavigator } from '@react-navigation/stack';
+import { createStackNavigator, CardStyleInterpolators } from '@react-navigation/stack';
 import HomePage from '../Components/HomePage';
 import DrillListPage from '../Components/DrillListPage';
 import DrillPage from '../Components/DrillPage';
 import DrillAnimationPage from '../Components/DrillAnimationPage';
+import Filters from '../Components/Filters';
 import TrainingListPage from '../Components/TrainingListPage';
 import TrainingPage from '../Components/TrainingPage';
 
 const Stack = createStackNavigator();
 
 export const Navigation = () => (
   <NavigationContainer>
-    <Stack.Navigator>
+    <Stack.Navigator screenOptions={{ cardStyleInterpolator: CardStyleInterpolators.forHorizontalIOS }}>
```

Comment:
What does your change ? 🤔

Reply:
Next screen comes from the left instead of the bottom, or something like this.

I'm not sure it makes a difference. That's from when I was trying some stuff to display filters differently.

Reply:
So yes, it makes new pages "come" from the right side of the screen instead of the bottom. What do you prefer?

(I have no idea what's best)

Reply:
I removed it, let's keep it simple